modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -92,6 +92,135 @@ using the `Flex` component and to replace the `spacing` prop with the `gap` prop
 > **Note**: If you are consuming `StackProps` for a custom component, please make sure to change the
 > `spacing` prop to the `gap` prop
 
+---
+
+### Menu in preview-react
```

Comment:
Add this as a menu item up above so that we can link to it

Reply:
I think this got removed when dealing with merge conflicts. I'll re-add